srcpkgs/giac/patches/pari_2_13.patch

```diff
@@ -0,0 +1,13 @@
+change in order to pass chk_fhan4 with pari-2.13
+
+--- a/src/pari.cc	2019-11-21 13:40:55.000000000 +0000
++++ b/src/pari.cc	2021-08-11 20:51:39.243502400 +0000
+@@ -88,7 +88,7 @@
+   static map<string,entree *> pari_function_table;
+   static void do_giac_pari_init(long maxprime){
+ #ifdef PARI_DYNAMIC_STACK
+-    long pari_mem_size=512000;
++    long pari_mem_size=1024000;
```

Comment:
Please try to upstream this ;)

Reply:
This is already in upstream. I'm working on an update of giac to 1.7.0-25 which won't need this patch.

Reply:
Added a note in the patch. We'll be removing that patch soon.